include/bluetooth/nrf/host_extensions.h

```diff
@@ -107,6 +107,15 @@ struct bt_conn_set_pcr_params {
  */
 int bt_conn_set_power_control_request_params(struct bt_conn_set_pcr_params *params);
 
+/** @brief Reduce the priority of the initiator when following AUX packets.
+ *
+ *  @param reduce Set to true to reduce the priority. Set to false to restore the default priority.
+ *
+ *  @return Zero on success or (negative) error code on failure.
+ *  @retval -ENOBUFS HCI command buffer is not available.
+ */
+int bt_nrf_host_extension_sdc_reduce_initator_aux_channel_priority(bool reduce);
```

Comment:
The other API in this file is not using `nrf_host`. Should remove that here as well? Maybe `bt_scan_reduce_initator_aux_channel_priority()`?

Reply:
ref discussion here:
https://github.com/nrfconnect/sdk-nrf/pull/15079#discussion_r1579594881